src/App.vue

```diff
@@ -38,10 +41,15 @@ export default {
   components: { MobileNavigation, LeftSection, RightSection },
   computed: {
     ...mapGetters('modals', ['opened']),
+    ...mapGetters(['isLoggedIn']),
     ...mapState(['address']),
     ...mapState('aeternity', ['sdk']),
   },
-  async created() {
+  async mounted() {
+    if (!this.isLoggedIn) {
+      this.$router.push({ name: 'landing' });
+    }
```

Comment:
I think this check should be done only by the index page because if somebody sends me a link to a specific post and I didn't use SW before then I steel should see the post instead of the landing page. Otherwise, we will break links for unregistered users.